src/Traits.h

```diff
@@ -1007,16 +1007,45 @@ struct Traits<std::chrono::system_clock::duration> : public Impl::CommonDuration
 template <typename ClockT, typename DurationT>
 struct Traits<std::chrono::time_point<ClockT, DurationT>> : public TraitsImpl<std::chrono::time_point<ClockT, DurationT>> {
     static std::string ToString(std::chrono::time_point<ClockT, DurationT> const &tp) {
-        std::ostringstream                  out;
-
-        date::operator <<(out, tp);
-        out.flush();
-
-        return out.str();
+        return date::format("%FT%TZ", date::floor<std::chrono::seconds>(tp));
     }
 
     static std::chrono::time_point<ClockT, DurationT> FromString(std::string const &value) {
-        std::ignore = value; throw std::logic_error("Not Implemented Yet");
+        std::string dash = "";
+        std::string colon = "";
+        std::string T = " ";
+        std::string Z = "Z";
+        std::string v(value);
+        if ((value.find_first_of("Z") == std::string::npos)) {

```

Comment:
It would be helpful to prefix this section of code with a comment that indicates that you are creating the date template based on values discovered in the incoming string.